memberscr.js

```diff
@@ -7,7 +7,10 @@ function on404(result){
     $("h3.memberprons").text("404 user not found");
 }
 
-function onGet(result){
+function onGet(result, textStatus, errorThrown){
+    if (errorThrown) {
+        $("h1").text(`There was an error! ${errorThrown}`);
```

Comment:
We could consider `return`ing inside of the `if (errorThrown)`, to prevent the rest of `onGet` from executing the rest of the function. This is good to go either way though.